packages/trace-viewer/src/ui/uiModeView.tsx

```diff
@@ -457,6 +488,31 @@ export const UIModeView: React.FC<{}> = ({
           requestedCollapseAllCount={collapseAllCount}
           setFilterText={setFilterText}
         />
+        <Toolbar noShadow={true} noMinHeight={true} className='settings-toolbar' onClick={() => setTestingOptionsVisible(!testingOptionsVisible)}>
```

Comment:
if we end up needing this more often, this should probably be abstracted into a `ToolbarSection` component.